components/VcItemActivationStatus.tsx

```diff
@@ -81,15 +80,15 @@ const WalletVerifiedActivationDetails: React.FC<
         <Text
           color={Theme.Colors.statusLabel}
           testID="activated"
-          weight="semibold"
+          weight="regular"
           size="smaller"
-          margin="10 10 10 10"
+          margin="8 10 10 5"
           style={
             !props.verifiableCredential
               ? Theme.Styles.loadingTitle
-              : Theme.Styles.subtitle
+              : Theme.Styles.statusLabel
           }
-          children={t('profileAuthenticated')}></Text>
+          children={t('closedCardProfileAuthenticated')}></Text>
```

Comment:
Is it necessary to add new key value ? we can change the value in local files(en,fil,hin..)

Reply:
needed to add since there are 2 different labels for closed  and open card activation labels.

Reply:
used the walletbinding label